README.md

```diff
@@ -65,19 +65,21 @@ Supported Compilers
+| CUDA (nvcc)                                                                    | :white_check_mark: <br/> (CUDA 11.0 - 12.0) | :white_check_mark: <br/> (CUDA 11.4 - 12.0) | :white_check_mark: <br/> (CUDA 12.0 - 12.1) | :x:                    | :white_check_mark: <br/> (CUDA 11.0-11.2; 11.6 - 12.0) | :white_check_mark: <br/> (CUDA 11.2, 11.6 - 12.0)       | :white_check_mark: <br/> (CUDA 11.6 - 12.0)      | :white_check_mark: <br/> (CUDA 11.7 - 12.0) | :white_check_mark: <br/> (CUDA 11.8 - 12.0)       | :white_check_mark: <br/> (CUDA 12.1)   | -                            | :x:                | :x:                                                   | :x:                                  |
+| CUDA (clang)                                                                   | -                                           | -                                           | -                                           | :x:                    | :x:                                                    | :x:                                                     | :x:                                              | :x:                                         | :white_check_mark: (CUDA 11.0 - 11.5)             | :white_check_mark: (CUDA 11.0 - 11.5)* | -                            | :x:                | -                                                     | -                                    |
+| [HIP](https://alpaka.readthedocs.io/en/latest/install/HIP.html) (clang)        | -                                           | -                                           | -                                           | :x:                    | :x:                                                    | :x:                                                     | :x:                                              | :x:                                         | :white_check_mark: (HIP 5.0 - 5.2)                | :white_check_mark: (HIP 5.3 - 5.4)     | :white_check_mark: (HIP 5.5) | :x:                | -                                                     | -                                    |
 
 Other compilers or combinations marked with :x: in the table above may work but are not tested in CI and are therefore not explicitly supported.
 
+\* due to a [compiler bug](https://github.com/llvm/llvm-project/issues/58491) in debug mode only release builds are supported
```

Comment:
The `*` is easily overlooked. Maybe we can add `:heavy_exclamation_mark:` (renders as :heavy_exclamation_mark:) instead? Both here and in the matrix.

Reply:
I can also do this in the clang-16 PR so we don't have to rerun the tests.

Reply:
I would prefer a classic foot notes. Is `[1]` a option?